exercises/pangram/stack.yaml

```diff
@@ -0,0 +1 @@
+resolver: lts-8.6
```

Comment:
> the current Stack snapshot

Yes, I guess that is ambiguous. Perhaps "the track's current Snack snapshot" would be more accurate.

> Starting with `stack.yaml`, this could evolve to a file that would generate the full exercise structure.

Nice idea. We could start simple with a _template directory like https://github.com/exercism/xjava/tree/master/exercises/_template or https://github.com/exercism/xkotlin/tree/master/exercises/_template